mongobin/getOrDownload.go

```diff
@@ -150,10 +166,7 @@ func GetOrDownloadMongod(urlStr string, cachePath string, logger *memongolog.Log
 
 		return "", fmt.Errorf("error writing mongod binary from %s to %s: %s", mongodTmpFile.Name(), mongodPath, renameErr)
 	}
-
-	logger.Infof("finished downloading mongod to %s in %s", mongodPath, time.Since(downloadStartTime).String())
-
-	return mongodPath, nil
+	return "", nil
```

Comment:
What is the meaning of this line? You are basically making the whole function call useless, 
if this was intentional explain this change, otherwise revert to the old line 

```
return mongodPath, nil
```

Reply:
No semantics were changed in this PR. This piece is simply a  refactor aimed and retaining the exact same semantics from earlier. This line does not make the function call useless at all, since the function is side-effecting by means of extracting the actual file to the filesystem at the right location. I think you may have mistaken this return value to be that of `GetOrDownloadMongod`, but it is not, it is of `saveFile`.

This first return value is used exclusively at line 108, for returning when the error is returned. I agree, we could change this to return just the error, as the first value is unused.

Reply:
Can you add the refactor of return values in the PR? After that i will merge